pkg/cmd/config.go

```diff
@@ -501,7 +501,7 @@ kind: KubeOneCluster
 name: {{ .ClusterName }}
 
 versions:
-  kubernetes: "{{ .KubernetesVersion }}"
+ kubernetes: "{{ .KubernetesVersion }}"
```

Comment:
This blank space is expected (identation should be 2).

```suggestion
  kubernetes: "{{ .KubernetesVersion }}"
```

Also, please add this new field to the example config.